packages/node_modules/@webex/plugin-meetings/test/utils/webex-config.js

```diff
@@ -56,7 +56,10 @@ Config.webex = {
     reconnection: {
       enabled: true
     },
-    enableRtx: true
+    enableRtx: true,
+    experimental: {
```

Comment:
remove this and we will handle the unified space meeting toggle enablement by default in a seperate PR